specification/hanaonazure/resource-manager/Microsoft.HanaOnAzure/preview/2017-11-03-preview/hanaonazure.json

```diff
@@ -194,6 +194,47 @@
           }
         }
       },
+      "delete": {
+        "tags": [
+          "HanaOnAzure"
+        ],
+        "operationId": "HanaInstances_Delete",
+        "summary": "Deletes a SAP HANA instance.",
+        "description": "Deletes a SAP HANA instance with the specified subscription, resource group, and instance name.",
+        "x-ms-examples": {
+          "Get properties of a HANA instance": {
+            "$ref": "./examples/HanaInstances_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/HanaInstanceNameParameter"
+          }
+        ],
+        "responses": {
+          "201": {
```

Comment:
Usually 201 is returned for an async operation, however this operation isn't marked as being async.  Is this expected?

Reply:
Yes, this operation is asynchronous; to mark the operation as async, do I just need to use `"x-ms-long-running-operation": true,` in the API spec?

Reply:
Yep!

Reply:
Done